irods/test/unicode_test.py

```diff
@@ -70,6 +71,20 @@ def tearDown(self):
         self.coll.remove(recurse=True, force=True)
         self.sess.cleanup()
 
+    def test_object_name_containing_unicode__318(self):
+        dataname = u"réprouvé"
+        homepath = "/{0.zone}/home/{0.username}".format(self.sess)
+        try:
+            ET_set( XML_Parser_Type.QUASI_XML, self.sess.server_version )
+            path = u"{homepath}/{dataname}".format(**locals())
+            self.sess.data_objects.create( path )
+        finally:
+            ET_set( XML_Parser_Type.STANDARD_XML )
+            self.sess.data_objects.unlink (path, force = True)
```

Comment:
the unlink works, regardless of the XML_Parser_Type?

Reply:
Well, what we are testing here is that handling of unicode names is working in QUASI_XML. That wasn't ever an issue with  standard XML.  But it is likely without the fix I wouldn't have been able to cleanup the test detritus in case of failure.  It's the underlying query that fails, so unlink wouldn't find the file it is supposed to delete....

Reply:
gotcha